src/utils.js

```diff
@@ -312,7 +314,7 @@ export function toIdSchema(schema, id, definitions) {
   return idSchema;
 }
 
-export function parseDateString(dateString) {
+export function parseDateString(dateString, includeTime = true) {
```

Comment:
I tried but it was weird; we're really *always* parsing datetimes here (the JSON date format always exposes hours, minutes and seconds), and it's just an export option I'm adding here.